installer/scripts/service_control

```diff
@@ -93,6 +101,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    warn_if_agent_not_onboarded
```

Comment:
Don't allow agent to try and start if we're not onboarded, an error will just result.